pubspec.yaml

```diff
@@ -7,7 +7,7 @@ environment:
   sdk: '>=1.0.0'
```

Comment:
The PR uses the null aware operator "??" that was introduced in Dart 1.12. Should this be considered in the SDK constraint?